lib/ReportHistoryStore.jsx

```diff
@@ -167,6 +239,42 @@ export default class ReportHistoryStore {
         return promise;
     }
 
+    /**
+     * Gets the history by reportActionID.
+     *
+     * @param {Number} reportID
+     * @param {Boolean} ignoreCache
+     *
+     * @returns {Deferred}
+     */
+    getByActionID(reportID, ignoreCache) {
+        const promise = new Deferred();
+
+        // Remove the cache entry if we're ignoring the cache, since we'll be replacing it later.
+        if (ignoreCache) {
+            delete this.cache[reportID];
+        }
+
+        // We'll poll the API for the un-cached history
+        const cachedHistory = this.cache[reportID] || [];
+        const lastHistoryItem = _.last(cachedHistory) || {};
+
+        this.API.Report_GetHistory({
+            reportID,
+            reportActionID: lastHistoryItem.reportActionID || 0,
```

Comment:
This parameter was there when i originally started writing the code but was removed very recently. I was hoping we would update the API to bring it back but since there is no immediately plan for it I'll remove it here. I'll also update the method name since `byActionID` won't make sense anymore.